### PR TITLE
BLD: Install requests using conda

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

**Comment by ElDeveloper** [comment]
This is ready for review and should clear the failing build in #724. It removes a version cap for Sphinx and installs requests using conda (to avoid a pip error).

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 skbio_3 = "scikit-bio >= 0.4.1"
 base = ["numpy >= 1.7", "scipy >= 0.17.0", "click", "pandas",
         skbio_2, "jinja2 >= 2.9", "future"]
-doc = ["Sphinx < 1.6", "sphinx-bootstrap-theme"]
+doc = ["Sphinx", "sphinx-bootstrap-theme"]
```
Comment (thermokarst) [inline setup.py]: We did not have a super smooth transition from Sphinx 1 -> 2 in QIIME 2, hope this goes better for you...

Comment (ElDeveloper) [reply setup.py]: Thanks for the heads up, after you posted this comment I looked again and realized that the travis file and setup.py files were out of sync. I fixed that and errors ensued. For now we'll keep it at <1.6. ⏳ 

This PR now only installs requests using conda, which should solve the broken build we found in #724.

**Comment by ElDeveloper** [comment]
Thanks!

**Pull request merged**